docs/roadmap.md

```diff
@@ -21,6 +21,7 @@ Codebase stewards review the roadmap monthly as part of our [backlog pruning ses
 
 ## Longer term
 
+* Become an ISO standard
```

Comment:
Should we link to the [community call where we discussed this](https://blog.publiccode.net/community%20call/2023/10/09/notes-from-community-call-5-october-2023.html)?